src/Factory.php

```diff
@@ -141,13 +141,25 @@ public function target($name)
             if (preg_match('#^https?://#', $file)) {
                 $files[] = new Remote($file);
             } else {
-                $path = $scanner->find($file);
-                if ($path === false) {
-                    throw new RuntimeException("Could not locate $file for $name in any configured path.");
+                if (preg_match('/(.*)(\/\*.*?)$/', $file, $matches)) {
```

Comment:
What would be a recursive glob pattern like? I've been looking into how to make glob behave recursively with no success, and most people around the web recommend creating an auxiliary recursive function with glob or making use of RecursiveDirectoryIterator class.